File: solucoes/2010/nivel jr/fase 2/js/dentista.js
var N, X, Y
var inicio = 0
var termino = 0 
var consultas = 0

scanf("%d", "N")

for(var i = 0; i < N; i++){
    scanf("%d%d", "X", "Y")
    if(X >= termino){
        inicio = X
        termino = Y
        consultas += 1
    }
}

printf("%d", consultas)

File: solucoes/2010/nivel jr/fase 2/py/dentista.py
N = int(input())
inicio, termino = 0, 0 
consultas = 0

for i in range(N):
    X, Y = map(int, input().split())

    if X >= termino:
        inicio = X
        termino = Y
        consultas += 1

print(consultas)

